docker-compose.yml

```diff
@@ -18,6 +18,16 @@ services:
       - SA_PASSWORD=localSAPassw0rd
     depends_on:
       - mssql
+  mssqlimport:
```

Comment:
Please combine / merge `mssqlimport` into the exisitng `mssqlinit` container.

There should be one container and script that either imports the bacpac (if present) or bootstraps a fresh DB

Reply:
Hello @mpern 

I can merge both `mssqlimport` and the `mssqlinit` into the same script. As a bacpac file is not a backup but a snapshot we need an already created database to import the content of the snapshot.

Reply:
Ah right!
I should have looked at your script more carefully.

In that case:
Always create db 
Import bacpac, if file is available (and not imported previously)

Reply:
Hello @mpern . I'm progressing on the requested changes. After testing, I've found that it is possible to import a bacpac without an already created database. Despite to that, I think it would be better to initialise brand new database anyway and then check if a bacpac is present to be imported. What do you think ?

Reply:
I'm still not vouching the idea of having the init and the import on the same docker-compose service.

I'm keeping both services apart for now. I will wait for your review for the changes I've just added.

Reply:
Fair point.
Let's keep using two different services for now.